recipe/meta.yaml

```diff
@@ -53,25 +66,98 @@ requirements:
+
+# # skipping these because we don't have xformers
+{% set tests_to_skip = tests_to_skip + " or test_xformers_single_forward_parity" %}
+
+# # skipping these because we don't have iopath
+{% set tests_to_skip = tests_to_skip + " or test_file_io_async" %}
+{% set tests_to_skip = tests_to_skip + " or test_xformers_single_forward_parity" %}
+
+# RuntimeError: Mask Type should be defined
+# According to issue, fixed on main, but not in 0.12.2. 
+# https://github.com/facebookresearch/fairseq/issues/4858#issuecomment-1313135905
+{% set tests_to_skip = tests_to_skip + " or test_alignment" %}
+{% set tests_to_skip = tests_to_skip + " or test_alignment_full_context" %}
+{% set tests_to_skip = tests_to_skip + " or test_cmlm_transformer" %}
+{% set tests_to_skip = tests_to_skip + " or test_insertion_transformer" %}
+{% set tests_to_skip = tests_to_skip + " or test_iterative_nonautoregressive_transformer" %}
+{% set tests_to_skip = tests_to_skip + " or test_mixture_of_experts" %}
+{% set tests_to_skip = tests_to_skip + " or test_multilingual_transformer" %}
+{% set tests_to_skip = tests_to_skip + " or test_nonautoregressive_transformer" %}
+{% set tests_to_skip = tests_to_skip + " or test_transformer" %}
+{% set tests_to_skip = tests_to_skip + " or test_transformer_cross_self_attention" %}
+{% set tests_to_skip = tests_to_skip + " or test_transformer_pointer_generator" %}
+{% set tests_to_skip = tests_to_skip + " or test_translation_multi_simple_epoch" %}
+{% set tests_to_skip = tests_to_skip + " or test_translation_multi_simple_epoch_dicts" %}
+{% set tests_to_skip = tests_to_skip + " or test_translation_multi_simple_epoch_no_vepoch" %}
+{% set tests_to_skip = tests_to_skip + " or test_translation_multi_simple_epoch_src_tgt_dict_spec" %}
+{% set tests_to_skip = tests_to_skip + " or test_activation_checkpointing_does_not_change_metrics" %}
+{% set tests_to_skip = tests_to_skip + " or test_activation_offloading_does_not_change_metrics" %}
+{% set tests_to_skip = tests_to_skip + " or test_mask_for_xformers" %}
+
+# requires examples, but skipping examples import due to segmentation fault from the size of examples
+{% set tests_to_skip = tests_to_skip + " or test_laser_lstm" %}
+{% set tests_to_skip = tests_to_skip + " or test_laser_transformer" %}
+{% set tests_to_skip = tests_to_skip + " or test_linformer_roberta_masked_lm" %}
+{% set tests_to_skip = tests_to_skip + " or test_linformer_roberta_regression_multiple" %}
+{% set tests_to_skip = tests_to_skip + " or test_linformer_roberta_regression_single" %}
+{% set tests_to_skip = tests_to_skip + " or test_linformer_roberta_sentence_prediction" %}
+{% set tests_to_skip = tests_to_skip + " or test_r4f_roberta" %}
+{% set tests_to_skip = tests_to_skip + " or test_waitk_checkpoint" %}
+{% set tests_to_skip = tests_to_skip + " or test_multilingual_translation_latent_depth" %}
+{% set tests_to_skip = tests_to_skip + " or test_roberta_incremental_decoder" %}
 
 test:
+  source_files:
+    - tests
+    - scripts
```

Comment:
what is it used from `scripts`?

Reply:
these directories are used all over the place from the tests because they contain assets that the tests used. Not the cleanest thing.

Reply:
Looks like specifically only `scripts/average_checkpoints.py` for a test. I'll change the line to only pull in `average_checkpoints.py`.

Reply:
thanks